package.json

```diff
@@ -51,6 +51,7 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
+    "fs": "^0.0.1-security",
```

Comment:
"fs" is a part of node's stdlib, so this is redundant

Reply:
That's what I thought but it wasn't working for me until I did `yarn add fs` but I will remove it.

Reply:
Removed: https://github.com/safe-global/safe-deployments/pull/228/commits/022ea896e8732cd3a0de55abec6a37b5d1741449